nixos/modules/services/networking/seafile.nix

```
@@ -213,7 +215,15 @@ in {
           installedMinor=$(cat "${seafRoot}/server-setup" | cut -d"-" -f1 | cut -d"." -f2)
           pkgMajor=$(echo "${cfg.seafilePackage.version}" | cut -d"." -f1)
           pkgMinor=$(echo "${cfg.seafilePackage.version}" | cut -d"." -f2)
-          if [ $installedMajor != $pkgMajor ] || [ $installedMinor != $pkgMinor ]; then
+
+          if [ $installedMajor == $pkgMajor ] && [ $installedMinor == $pkgMinor ]; then
```

```suggestion
          if [[ $installedMajor == $pkgMajor && $installedMinor == $pkgMinor ]]; then
```
== should be undefined in [ ... ]